content/docs/dvclive/api-reference/live/log_params.md

```diff
@@ -0,0 +1,54 @@
+# Live.log_params()
+
+Logs the given group of parameters `params`, saving them as yaml.
+
+```py
+ def log_params(params: Dict[ParamLike]):
+```
+
+Also see `Live.log_param()`.
+
+## Usage
+
+```py
+from dvclive import Live
+
+live = Live()
+
+params = {
+    "lr": 1e-3,
+    "n_iter": 50
+}
```

Comment:
What do you think about including a nested param value here?

Reply:
Yeah, that's fine

Reply:
How about something like this? It showcases quite a few possibilities, although it's a bit large snippet

```python
params = {
    "num_classes": 10,
    "metrics": ["accuracy", "mae"],
    "training": {
        "epochs": 42,
        "batch_size": 128,
        "optimizer": {
            "name": "adam",
            "loss": "categorical_crossentropy",
            "learning_rate": 1e-3,
        },
    },
}
```